docs/security.md

```diff
@@ -0,0 +1,121 @@
+# Security
+
+The OpenTelemetry Collector defaults to operating in a secure manner, but is
+configuration driven. This document captures important security aspects and
+considerations for the Collector.
+
+## Configuration
+
+The Collector does not contain an embedded or default configuration and MUST
```

Comment:
Actually... the docker image does come with a default configuration file, and there's an issue about adding a default config file to the collector: https://github.com/open-telemetry/opentelemetry-collector/issues/886

Reply:
Agreed -- my point was the binary does not have an embedded one today though packaging may include one. Also, this should change in the future and then this doc needs to be updated. I figured reflecting the current world made sense. Let me try to clarify and even link to known next steps so people are aware